src/hotspot/share/ci/ciField.cpp

```diff
@@ -250,6 +250,10 @@ static bool trust_final_non_static_fields(ciInstanceKlass* holder) {
   return TrustFinalNonStaticFields;
 }
 
+static bool trust_final_non_static_fields_of_type(Symbol* signature) {
+  return signature == vmSymbols::java_lang_StableValue_signature();
```

Comment:
Just a note that we will need to decide whether to keep this or not...

Reply:
We might change this to require stable values to be strict final instead if strict final is previewed at the same time as stable values - https://openjdk.org/jeps/8350458

Reply:
This has now been removed.